### DISCUSSION TITLE
created fork, but it doesn't work with the revanced manager

### DISCUSSION THREAD

**Discussion by ggggrand** [ReVanced/revanced-discussions#1029] [Question]
i set up everything, 
deleted the old forks of the patches and integrations, created new ones, 
but it doesn't work.
I still can't get the patches to work, sometimes they would show up, but the app can't be patched, revanced manager gets stuck in patching with a blank screen.
Please help, I have never done anything like this, so please explain like I'm a noob.

**Comment by oSumAtrIX** [comment]
You can find steps to use ReVanced Manager to patch apps here: https://github.com/ReVanced/revanced-manager/tree/dev/docs

**Reply by ggggrand** [reply]
i already changed the patches organization and integrations organization in the sources. 
What else is required?

**Reply by oSumAtrIX** [reply]
Release the patches and integrations correctly

**Reply by ggggrand** [reply]
what do you mean? it is released.

**Reply by oSumAtrIX** [reply]
No you havent:

![image](https://github.com/ReVanced/revanced-discussions/assets/13122796/326604e3-4e8e-46dd-87e3-2814d8221a82)

**Reply by oSumAtrIX** [reply]
Follow the template's instructions to setup your repository properly: https://github.com/ReVanced/revanced-patches-template#-get-started

**Reply by ggggrand** [reply]
All I wanted was to change one or two patches, not create new ones. 
I've already done the changes, but manager can't patch

**Comment by LisoUseInAIKyrios** [comment]
Are you making changes just for yourself?

If so then you don't need Manager.  Build locally and use CLI tools to patch and install.
